bindings/pyroot/pythonizations/src/TTreePyz.cxx

```diff
@@ -142,8 +142,18 @@ static PyObject *WrapLeaf(TLeaf *leaf)
 }
 
 // Allow access to branches/leaves as if they were data members
-PyObject *GetAttr(PyObject *self, PyObject *pyname)
+// Returns a Python tuple where the first element is either the desired
+// CPyCppyy proxy, or an address that still needs to be wrapped by the caller
+// in a proxy using cppyy.ll.cast. In the latter laster, the second tuple
```

Comment:
I am unsure about this, but maybe you meant
```suggestion
// in a proxy using cppyy.ll.cast. In the latter case, the second tuple
```

Reply:
My thoughts were out-of-sync with the typing :slightly_smiling_face: